$NetBSD$

Fix OUI_PATH. Note that this gets SUBST'ed, so /usr/local/etc is fine.

--- scripts/airodump-ng-oui-update.orig	2013-04-16 20:12:28.000000000 +0000
+++ scripts/airodump-ng-oui-update
@@ -4,19 +4,7 @@ CURL=`which curl 2>/dev/null`
 WGET=`which wget 2>/dev/null`
 OUI_DOWNLOAD_URL="http://standards.ieee.org/develop/regauth/oui/oui.txt"
 
-OUI_PATH0="/etc/aircrack-ng"
-OUI_PATH1="/usr/local/etc/aircrack-ng"
-OUI_PATH2="/usr/share/aircrack-ng"
-if [ -d "$OUI_PATH0" ]; then
-	OUI_PATH="$OUI_PATH0"
-elif [ -d "$OUI_PATH1" ]; then
-	OUI_PATH="$OUI_PATH1"
-elif [ -d "$OUI_PATH2" ]; then
-	OUI_PATH="$OUI_PATH2"
-else
-	# default
-	OUI_PATH="$OUI_PATH0"
-fi
+OUI_PATH="/usr/local/etc/aircrack-ng"
 
 AIRODUMP_NG_OUI="${OUI_PATH}/airodump-ng-oui.txt"
 OUI_IEEE="${OUI_PATH}/oui.txt"
